test/unit/product/product-component.js

```diff
@@ -1236,6 +1241,7 @@ describe('Product Component class', () => {
               src: firstImage.src,
               srcLarge: expectedSrcLarge,
               srcOriginal: firstImage.src,
+              altText: (firstImage.altText || 'test'),
```

Comment:
I removed the `||` and left, `'test'`. A bit literal but I always think thats good to do when testing. Moreover, I found `firstImage.altText` was null so the tests would fail if I chose `firstImage.altText`.